### PR TITLE
Set error message for where to report bugs

### LINKED ISSUE #11586: Fix reduction_v2_uneven.mlir test after integration
https://github.com/iree-org/iree/pull/11576 disables reduction_v2_uneven.mlir to make forward progress.

### PR DESCRIPTION
**Pull request by jpienaar**
Avoids recommending filing bugs with LLVM project for IREE project tooling.

### CODE REVIEW

**Comment by benvanik** [comment]
uh, that's a lot of commits? 😮

**Comment by benvanik** [comment]
(otherwise LGTM - just need to clean up whatever happened in git - thanks for doing this!)

**Comment by jpienaar** [comment]
> uh, that's a lot of commits? open_mouth

I don't know why I keep doing this ... I git pull main && git branch && git checkout && git commit && gh pr create ... I should just explicitly have a git fetch gh && git rebase gh/main in there too.

**Comment by benvanik** [comment]
if you pull main while on main then it should rebase for you - maybe you're issuing your git pull main while on another branch?
my flow is usually:
```
git checkout main && git pull --prune
git checkout -b new-branch
<work>
```

**Review by benvanik — APPROVED** [review]

File: tools/iree-opt-main.cc [hunk 1]
```diff
@@ -9,5 +9,11 @@
 // Based on mlir-opt but registers the passes and dialects we care about.
 
 #include "iree/compiler/API2/ToolEntryPoints.h"
+#include "llvm/Support/PrettyStackTrace.h"
 
-int main(int argc, char **argv) { return ireeOptRunMain(argc, argv); }
+int main(int argc, char **argv) {
+  llvm::setBugReportMsg(
```
Comment (benvanik) [inline tools/iree-opt-main.cc:15]: this calls the method in IREEOptToolEntryPoint.cpp so it shouldn't be needed here

**Pull request merged**